pkg/apiserver/api_installer.go

```diff
@@ -104,9 +104,9 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	admit := a.group.Admit
 	context := a.group.Context
 
-	serverGroupVersion := a.group.GroupVersion
+	externalGroupVersion := a.group.GroupVersion
 	if a.group.ServerGroupVersion != nil {
-		serverGroupVersion = *a.group.ServerGroupVersion
+		externalGroupVersion = *a.group.ServerGroupVersion
```

Comment:
I don't get why`externalGroupVersion` is a better name. To me, serverGroupVersion makes more sense since it's server picks to use version.